lib/final-api/v1/http/ddtf_build.rb

```diff
@@ -70,13 +70,12 @@ def test_data
             'stoppedBy': build.stopped_by.try(:name), # TODO
 
             'isTsd': true,
-            'checkpoints':    config[:checkpoints],
-            'debugging':      config[:debbuging],
-            'buildSignal':    config[:build_signal],
-            'scenarioScript': config[:scenario_script],
-            'packageSource':  config[:package_source],
+            'checkpoints':    checkpoints,
+            'debugging':      config[:debbuging] || false,
```

Comment:
I remember Zdeněk Novotný already fixed it as part of another issue, so I am surprised it is still here